src/main/webapp/app/pages/userManagement/UserDetailsPage.tsx

```diff
@@ -46,15 +56,46 @@ export default class UserDetailsPage extends React.Component<{
     try {
       // Hard code the max returned user size. Need to fix pagination issue.
       this.users = await client.getAllUsersUsingGET({ size: 2000 });
+      // Display all commerical users by default
+      this.toggleFilter(USER_BUTTON_TYPE.COMMERCIAL);
       this.loadedUsers = true;
     } catch (e) {
       notifyError(e, 'Error fetching users');
     }
   }
 
   @action
-  toggleFilter(query: any) {
-    this.currentSelectedFilter = query;
+  toggleFilter(button: string) {
+    this.currentSelectedButton = button;
+    switch (this.currentSelectedButton) {
```

Comment:
Do we need the switch? Seems like a if-else condition

Reply:
I think it's clear and easy for extending filters in the future and switch is also good for if-else conditions more than 3.

Reply:
I would not worry about the extension when the requirement is not clear yet. At this point they are dup code.